### PR TITLE
adding appveyor.yml

### LINKED ISSUE #19: AppVeyor
Goal is to add AppVeyor CI framework into project

### PR DESCRIPTION
**Pull request by conztantinee**
committed build files can be ignored during review

### CODE REVIEW

**Comment by conztantinee** [comment]
can anyone the script appveyor.yml script and give me feedback?

**Comment by SubhanHakverdiyev** [comment]
Can you please not commit the build files? You can add them to gitignore or unversioned files

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+test_script:
+  - cmd: gradlew.bat test --info
+
+cache:
+  - '%USERPROFILE%\.gradle\caches'
+  - '%USERPROFILE%\.gradle\wrapper'
+
+notifications:
+  - provider: Email
```
Comment (SubhanHakverdiyev) [inline appveyor.yml]: I think we don't need Email or need to specify because now it is giving error as email is **`null`**

Comment (conztantinee) [reply appveyor.yml]: You're right. Actually, notifications part can be emitted wholely

**Comment by ayazbekobbelek** [comment]
LGTM!

**Review by ayazbekobbelek — APPROVED** [review]

**Pull request merged**